packages/react-theme-sass/sass/Sass.test.ts

```diff
@@ -0,0 +1,15 @@
+import { renderSync } from 'node-sass';
+import { webLightTheme } from '@fluentui/react-theme';
+
+describe('Sass tokens', () => {
+  it('exist for all theme tokens', () => {
+    const data = [
+      '@import "sass/tokens.scss";',
+      ...Object.keys(webLightTheme)
+        .filter(t => !t.startsWith('colorPalette')) // FIXME
```

Comment:
This line will go away once the shared color tokens will be added (before merging).